src/odemis/acq/stream/_static.py

```diff
@@ -935,3 +935,52 @@ def onSpectrumBandwidth(self, value):
         """
         self._updateHistogram()
         self._shouldUpdateImage()
+
+
+class RGBUpdatableStream(StaticStream):
```

Comment:
I inherit from StaticStream because the functions _init_projection_vas and _init_thread are overwritten with an empty function in RGBStream, but these functions are needed to properly update the image.